relay-metrics/src/bucket.rs

```diff
@@ -739,6 +739,20 @@ impl CardinalityItem for Bucket {
     }
 }
 
+/// Utility that can wrap a bucket and inspect its contents.
+pub struct BucketInspector<'a>(pub &'a Bucket);
```

Comment:
We already need to identify the usage metrics in more places (related to rate limiting, should be contained to `relay-server/metrics` module, e.g. `TrackableBucket`), maybe we can unify this.